### DISCUSSION TITLE
GitHub account hacked

### DISCUSSION THREAD

**Discussion by AnuPlayz** [community/community#129746] [Other Feature Feedback, Questions, & Ideas]
My GitHub has been hacked idk how, one morning I recieve email that my password changed and both emails are removed now... I tried contacting support but no help yet ticket #2853706

What should I do?![Screenshot_2024-06-24-08-54-05-98_e307a3f9df9f380ebaf106e1dc980bb6.jpg](https://github.com/community/community/assets/120038186/30fd8251-422d-49d5-bb89-1f3253d041c9)

Edit : my phone didn't got logged out so I can still message here but I don't have access to my account. HELPPP

**Comment by ebndev** [comment]
Hi @AnuPlayz,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
